src/common/utils/gpo.h

```diff
@@ -115,6 +118,53 @@ namespace powertoys_gpo {
         }
     }
 
+    inline std::string getPolicyListValue(const std::wstring& registry_list_path, const std::wstring& registry_list_value_name)
+    {
+        // This function returns the value of an entry of an policy list. The user scope is only checked, if the list is not enabled for the machine to not mix the lists. If there is no value or no list is found in the registry we return an empty string.
+
+        HKEY key{};
+        DWORD reg_value_type = REG_SZ;
+        char* string_value{};
+        DWORD string_value_length = sizeof(char*);
```

Comment:
@jaimecbernardo 
Is using `char*` the correct way of doing it for reading a REG_SZ?

Btw, the GPO and setting ui parts are ready and working. (I still have to implement the wox part.) So if you are interested, you can try it out. 😉

Reply:
As this is causing a crash the answer is no.